src/common/config.cc

```diff
@@ -1156,6 +1169,8 @@ Option::value_t md_config_t::_expand_meta(
 	out += stringify(getpid());
       } else if (var == "cctid") {
 	out += stringify((unsigned long long)this);
+      } else if (var == "home") {
+	out = string(getenv("HOME"));
```

Comment:
Small issue: getenv() can return NULL, and std:;string crashes when passed null.  Change to
```
const char *home = getenv("HOME");
out = home ? std::string(home) : std::string();
```
or similar

Reply:
Done.